### PR TITLE
[TRANSLATION] Update Romanian translation

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose

JIRA issue: [CORE-15023](https://jira.reactos.org/browse/CORE-15023)

Based on patch by @reactos-contributor.

### CODE REVIEW

File: base/setup/usetup/lang/ro-RO.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* �tefan Fulea (stefan dot fulea at mail dot com) */
+/* Ştefan Fulea (stefan dot fulea at mail dot com) */
```
Comment (HBelusca) [inline base/setup/usetup/lang/ro-RO.h]: What is changing there?

Comment (Fraizeraust) [reply base/setup/usetup/lang/ro-RO.h]: I highly suspect the GitHub website forcibly changed the encoding type of the file to UTF-8. Looks like Katayama committed these patches in GitHub than using the command line prompt.

Comment (katahiromz) [reply base/setup/usetup/lang/ro-RO.h]: I will revert file "base/setup/usetup/lang/ro-RO.h".

Comment (HBelusca) [reply base/setup/usetup/lang/ro-RO.h]: For reminders, **NEVER EVER** use the GitHub website editor to manipulate localized resource files!

Comment (katahiromz) [reply base/setup/usetup/lang/ro-RO.h]: Reverted. And then I will apply ro-RO.h partically...

Comment (katahiromz) [reply base/setup/usetup/lang/ro-RO.h]: OK, done.

File: base/setup/usetup/lang/ro-RO.h [hunk 2]
```diff
@@ -1832,7 +1832,7 @@ MUI_STRING roROStrings[] =
     {STRING_REBOOTCOMPUTER2,
     "   ENTER = Repornire calculator"},
     {STRING_REBOOTPROGRESSBAR,
-    " Calculatorul va reporni �n %li secunde... "},
+    " Calculatorul va reporni �n %li seconde... "},
```
Comment (Fraizeraust) [inline base/setup/usetup/lang/ro-RO.h]: "secunde" not "seconde".

**Comment by katahiromz** [comment]
Everything done.

File: base/setup/usetup/lang/ro-RO.h (hunk 2 above)
Comment (katahiromz) [reply base/setup/usetup/lang/ro-RO.h]: OK.

**Comment by binarymaster** [comment]
Now it needs to be squashed, and amended with correct author...

`git commit --amend --no-edit --author "Ștefan Fulea <stefan.fulea@mail.com>"`

**Comment by katahiromz** [comment]
Done.

**Comment by HBelusca** [comment]
It hasn't been squashed.

**Comment by katahiromz** [comment]
Whap! Repository was broken... :(

**Comment by Fraizeraust** [comment]
```
git checkout romanian-update
git rebase -i HEAD~X // The X number of commits you have in the branch (including the offending one)
git push origin romanian-update --force-with-lease
```
When you type the second command which will do an interactive base, it'll open a text editor with the list of commits you have in the branch. It should be the `f0ca65a` commit there as well.

Since `f0ca65a` is completely unrelated within this PR, we've to remove it. To do so you must replace the `pick` with `drop`. In order to squash the commits as well you just need to replace the "pick" of other commits with "squash" EXCEPT the first commit of the branch, which, in this case is `[TRANSLATION] Update Romanian translation` with commit ID `f2c11b3`.

When you close the text editor, Git will open another which wants you to choose which name you want for the commit. You can freely keep the first name of the commit as is. When you're done, you must do a force push by executing the last command line at the top. It should squash all the commits now.

**Comment by Fraizeraust** [comment]
The Fulea's patch is a bit messed up. I don't know for which reason he did touch the string in USETUP which it has been already translated in [c1bf18b](https://github.com/reactos/reactos/commit/c1bf18bada6bda1b4f691efd9383c3380292411d) (when Hermes Belusca merged my PR). Now we have a merge conflict...

@katahiromz Looking at your forked repository, you have 69 commits behind master. In order to fix merge conflicts you've to sync your fork with the latest commits merged in master. I assume you already have the knowledge how to do so, if not then I can help you on this.

When you have updated your fork, we must also update the `romanian-update` branch too. To do so, type the command lines in the following order:
```
git checkout romanian-update
git fetch origin // This will get the latest refs and objects from origin (local repository)
git merge origin/master // Merge into your branch
git push --force-with-lease // Do a force push
```

**Comment by katahiromz** [comment]
@reactos-contributor What's your e-mail address?

**Comment by reactos-contributor** [comment]
stefan.fulea@mail.com
(ignore the programer.net domain alias)

**Comment by reactos-contributor** [comment]
"@reactos-contributor I suppose you weren't aware of #847 when it has been created?"
That's right.

**Comment by katahiromz** [comment]
Done.

**Pull request merged**